clients/src/main/java/org/apache/kafka/clients/admin/QuorumInfo.java

```diff
@@ -29,19 +32,22 @@ public class QuorumInfo {
     private final long highWatermark;
     private final List<ReplicaState> voters;
     private final List<ReplicaState> observers;
+    private final List<Node> nodes;
 
     QuorumInfo(
         int leaderId,
         long leaderEpoch,
         long highWatermark,
         List<ReplicaState> voters,
-        List<ReplicaState> observers
+        List<ReplicaState> observers,
+        List<Node> nodes
```

Comment:
It seems better to have a `Map<Integer, Node>` since we know the IDs are unique.

Reply:
Fixed.